server/.env

```diff
@@ -0,0 +1,2 @@
+ACCESS_TOKEN_SECRET=628c1f0165007a86b31d9c62ba5320d3aac1fc9363ec16f1fd8009be26f683aecadb39627e607bd80c37dc82419074575cc1234996e3e39d06494d580d9d8be
+REFRESH_TOKEN_SECRET=00ba6fad2d89be129e94235bd5ca6a123c55415b521f882d3951c2ec54924907b0b7262a53f8c042c2aee8b25946d5f8bd6c26b698dbfc83dfe167923c175548
```

Comment:
We should not be keeping our secrets public on Github :) These should be managed on a system-by-system basis. You can store them locally in your `.bash_profile` so they load every time you open Terminal. Once we set the app up to run remotely, we'll need to securely configure these values for that server as well.

tldr; this file should be removed